rai/blockstore.cpp

```diff
@@ -896,7 +896,7 @@ void rai::block_store::frontier_del (MDB_txn * transaction_a, rai::block_hash co
 size_t rai::block_store::frontier_count (MDB_txn * transaction_a)
```

Comment:
Since state-block opened accounts are not included in the frontier table, the function should probably be renamed `account_count`.

`TEST (block_store, frontier_count)` => `TEST (block_store, account_count)` as well. 

But leave the RPC action name intact.

Reply:
I'd prefer to keep the function name consistent with the RPC action.

Reply:
Hm, not sure why the block store should care what RPC actions are (potentially wrongly) called, but I guess in this case there are no other users of the function beyond the test.

Reply:
Oops, I'm on mobile and I thought this function was the rpc service function :P

Yeah renaming it's a good idea.